python/docs/source/development/contributing.rst

```diff
@@ -120,7 +120,7 @@ Prerequisite
 
 PySpark development requires to build Spark that needs a proper JDK installed, etc. See `Building Spark <https://spark.apache.org/docs/latest/building-spark.html>`_ for more details.
 
-Note that if you intend to contribute to Spark Connect in Python, ``buf`` version ``1.28.1`` is required, see `Buf Installation <https://docs.buf.build/installation>`_ for more details.
+Note that if you intend to contribute to Spark Connect in Python, ``buf`` version ``1.29.0`` is required, see `Buf Installation <https://docs.buf.build/installation>`_ for more details.
```

Comment:
probably we can remove the specified version here

Reply:
Perhaps we need to suggest it in the documentation to facilitate local development, otherwise it may cause inconvenience.